requirements.txt

```diff
@@ -2,3 +2,4 @@ requests==2.18.4
 jsonobject>=0.8.0
 PyYAML==3.12
 gevent==1.2.2
+argparse>=1.4
```

Comment:
I just added this to avoid the warning when installing commcare-cloud:
```
commcare-cloud 0.0.0 has requirement argparse>=1.4, but you'll have argparse 1.2.1 which is incompatible.
```